pandas/tests/series/methods/test_interpolate.py

```diff
@@ -97,6 +99,22 @@ def test_interpolate_time_raises_for_non_timeseries(self):
         with pytest.raises(ValueError, match=msg):
             non_ts.interpolate(method="time")
 
+    @td.skip_if_no_scipy
+    def test_interpolate_cubicspline(self):
+
+        ser = Series([10, 11, 12, 13])
+
+        expected = Series(
+            [11.00, 11.25, 11.50, 11.75, 12.00, 12.25, 12.50, 12.75, 13.00],
+            index=Index([1.0, 1.25, 1.5, 1.75, 2.0, 2.25, 2.5, 2.75, 3.0]),
+        )
+        # interpolate at new_index
+        new_index = ser.index.union(Index([1.25, 1.5, 1.75, 2.25, 2.5, 2.75])).astype(
+            float
+        )
+        interp_s = ser.reindex(new_index).interpolate(method="cubicspline")
```

Comment:
Could you rename this `result` and do the _getitem_ here so the next line could be `tm.assert_series.equal(result, expected)`?

Reply:
Done, thanks!